priv/app.config

```diff
@@ -1,5 +1,9 @@
 [
- {amoc, [{interarrival, 75}]},
+ {amoc, [{interarrival, 75},
+         {pubsub_nodes, 5},
```

Comment:
I think it's not a good idea to put scenario specific config in the repository. If we need to read it from the app config I'd rather set default values in the scenario when reading them.